src/components/LoadingShape/styled/index.js

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import AvatarIcon from '../icons/avatar';
+import ImageIcon from '../icons/image';
+
+const StyledShapeContainer = styled.div`
+    position: relative;
+    width: 100%;
+    height: 100%;
+`;
+
+const StyledImageIcon = styled(ImageIcon)`
+    ${props =>
```

Comment:
I think we should add the color `backgroundMain` on the SVG

Reply:
Fixed